scispacy/candidate_generation.py

```diff
@@ -384,7 +384,11 @@ def create_tfidf_ann_index(
     # Default values resulted in very low recall.
 
     # set to the maximum recommended value. Improves recall at the expense of longer indexing time.
-    # TODO: This variable name is so hot because I don't actually know what this parameter does.
+    # We use the HNSW (Hierarchical Navigable Small World Graph) representation which is constructed
+    # by consecutive insertion of elements in a random order by connecting them to M closest neighbours
+    # from the previously inserted elements. These later become bridges between the network hubs that 
+    # improve overall graph connectivity. (bigger M -> higher recall, slower creation)
+    # For more details see: https://arxiv.org/ftp/arxiv/papers/1603/1603.09320.pdf
```

Comment:
Any reason for the long url instead of https://arxiv.org/pdf/1603.09320.pdf?

Reply:
Nope, I just copypasted from my issue. Fixed now